Library/Homebrew/utils/pypi.rb

```diff
@@ -97,8 +97,8 @@ def pypi_info(new_version: nil)
     sig { returns(String) }
     def to_s
       if valid_pypi_package?
-        out = name
-        out += "[#{extras.join(",")}]" if extras.present?
+        out = T.must(name)
```

Comment:
If we're doing a `T.must` here feels like it might be nice to enforce in `def name` and `def initialize` that it's a `String` type instead?

Reply:
Still think this might be nice but not a blocker.

Reply:
Either I'm being dense or Sorbet is being weird, gonna punt on this for it to be future Issy (and/or Mike, or anyone else)'s problem.